src/sinks/gcp/pubsub.rs

```diff
@@ -213,6 +213,7 @@ mod tests {
             r#"
            project = "project"
            topic = "topic"
+           skip_authentication = false
```

Comment:
Please revert this, this test failure indicates that the configuration structure change has caused breakage. We do not want to fix that by changing the test.

I missed that there was no `#[serde(default)]` on the structure or field. Instead of the derivative bits, you would need `#[serde(default = "default_skip_authentication")]` above the field, and then a new `fn default_skip_authentication() -> bool` that returns that default value.

Reply:
ah, serde is the serialisation framework, thank you. done